dockerfiles/evmapp/entrypoint.sh

```diff
@@ -90,56 +100,72 @@ done
 
 if [ "${SCNODE_FORGER_RESTRICT:-}" = "true" ]; then
   if [ -z "${SCNODE_ALLOWED_FORGERS:-}" ]; then
-    echo "Error: Environment variable SCNODE_ALLOWED_FORGERS required when SCNODE_FORGER_RESTRICT=true."
+    echo "Error: Environment variable SCNODE_ALLOWED_FORGERS is required when SCNODE_FORGER_RESTRICT=true."
     sleep 5
     exit 1
   fi
 fi
 
-SCNODE_REMOTE_KEY_MANAGER_ENABLED="${SCNODE_REMOTE_KEY_MANAGER_ENABLED:-false}"
-export SCNODE_REMOTE_KEY_MANAGER_ENABLED
-
+# Cert signing
 if [ "${SCNODE_CERT_SIGNING_ENABLED:-}" = "true" ]; then
-  if [ -z "${SCNODE_CERT_SIGNERS_SECRETS:-}" ] && [ "${SCNODE_REMOTE_KEY_MANAGER_ENABLED:-}" != "true" ]; then
-    echo "Error: When SCNODE_CERT_SIGNING_ENABLED=true either environment variable SCNODE_CERT_SIGNERS_SECRETS is required or remote key manager SCNODE_REMOTE_KEY_MANAGER_ENABLED=true is required."
-    sleep 5
-    exit 1
-  fi
-
-  if  [ "${SCNODE_REMOTE_KEY_MANAGER_ENABLED:-}" = "true" ] && [ -z "${SCNODE_REMOTE_KEY_MANAGER_ADDRESS:-}" ]; then
-    echo "Error: When SCNODE_CERT_SIGNING_ENABLED=true and SCNODE_REMOTE_KEY_MANAGER_ENABLED=true SCNODE_REMOTE_KEY_MANAGER_ADDRESS needs to be set."
-    sleep 5
-    exit 1
-  fi
-
-  if  [ "${SCNODE_REMOTE_KEY_MANAGER_ENABLED:-}" = "true" ] && [ -z "${SCNODE_REMOTE_KEY_MANAGER_REQUEST_TIMEOUT:-}" ]; then
-    echo "Error: When SCNODE_CERT_SIGNING_ENABLED=true and SCNODE_REMOTE_KEY_MANAGER_ENABLED=true SCNODE_REMOTE_KEY_MANAGER_REQUEST_TIMEOUT needs to be set."
-    sleep 5
-    exit 1
-  fi
-
-  if  [ "${SCNODE_REMOTE_KEY_MANAGER_ENABLED:-}" = "true" ] && [ -z "${SCNODE_REMOTE_KEY_MANAGER_PARALLEL_REQUESTS:-}" ]; then
-    echo "Error: When SCNODE_CERT_SIGNING_ENABLED=true and SCNODE_REMOTE_KEY_MANAGER_ENABLED=true SCNODE_REMOTE_KEY_MANAGER_PARALLEL_REQUESTS needs to be set."
-    sleep 5
-    exit 1
+  # Checking first for either SCNODE_CERT_SIGNERS_SECRETS not empty and SCNODE_REMOTE_KEY_MANAGER_ENABLED !=true and vice versa
+  if [ -z "${SCNODE_CERT_SIGNERS_SECRETS:-}" ]; then
+    if [ "${SCNODE_REMOTE_KEY_MANAGER_ENABLED:-}" != "true" ]; then
+      echo "Error: if SCNODE_CERT_SIGNING_ENABLED=true either SCNODE_CERT_SIGNERS_SECRETS must NOT be empty or SCNODE_REMOTE_KEY_MANAGER_ENABLED=true is required to be set."
+      sleep 5
+      exit 1
+    fi
+  else
+    if [ "${SCNODE_REMOTE_KEY_MANAGER_ENABLED:-}" = "true" ]; then
+      echo "Error: if SCNODE_CERT_SIGNING_ENABLED=true and SCNODE_CERT_SIGNERS_SECRETS is NOT empty, then SCNODE_REMOTE_KEY_MANAGER_ENABLED must be set to false."
+      sleep 5
+      exit 1
```

Comment:
I think this is not true. A node can use both local keys (from config file) and remote_key_manager. It will check both.

@ysibirski @lander86 
```
- Given SCNODE_CERT_SIGNING_ENABLED = True

  - SCNODE_CERT_SIGNERS_SECRETS **not set** AND SCNODE_REMOTE_KEY_MANAGER_ENABLED **False** ====> ERROR
  
  - SCNODE_CERT_SIGNERS_SECRETS **set** AND SCNODE_REMOTE_KEY_MANAGER_ENABLED **False** ====> VALID configuration
  
  - SCNODE_CERT_SIGNERS_SECRETS **not set** AND SCNODE_REMOTE_KEY_MANAGER_ENABLED **True** ====> VALID configuration
  
  - SCNODE_CERT_SIGNERS_SECRETS **set** AND SCNODE_REMOTE_KEY_MANAGER_ENABLED **True** ====> VALID configuration
  
- Depending on which option we use more env vars might be required to be set:

  - SCNODE_REMOTE_KEY_MANAGER_ENABLED =True
  
    - SCNODE_REMOTE_KEY_MANAGER_ADDRESS ===> Mandatory
    
    - SCNODE_REMOTE_KEY_MANAGER_REQUEST_TIMEOUT ===> Mandatory
    
    - SCNODE_REMOTE_KEY_MANAGER_PARALLEL_REQUESTS ===> Mandatory
    

```
@cronicc correct me if I'm wrong

So with the changes in this PR the fourth case (SCNODE_CERT_SIGNERS_SECRETS set AND SCNODE_REMOTE_KEY_MANAGER_ENABLED True ====> VALID configuration) is not covered. 

I think we just need to export the new env vars (export REMOTE_KEY_MANAGER_REQUEST_TIMEOUT and export REMOTE_KEY_MANAGER_PARALLEL_REQUESTS) without changing the if-else logic.

Reply:
```
SCNODE_CERT_SIGNERS_SECRETS **set** AND SCNODE_REMOTE_KEY_MANAGER_ENABLED **True** ====> VALID configuration
```
I did not know about this scenario. I was thinking it can not be `set` and `true` at the same time

Reply:
This can be tested using the secure enclave. Setting some keys in the config file and some in the secure enclave. You can see there are calls for signatures to the enclave in the logs.
I'm pretty sure we did that test at some point, and unless the node has been changed it should still work.

Reply:
@cronicc and @otoumas I just pushed the fix. Please take a look. It will allow now the following scenario to pass

> SCNODE_CERT_SIGNERS_SECRETS set AND SCNODE_REMOTE_KEY_MANAGER_ENABLED True ====> VALID configuration